microsoft-edge/devtools-guide-chromium/security/index.md

```diff
@@ -44,7 +44,7 @@ Use the **Security** panel to inspect the security of a page.
 
```

Comment:
i consider non-blocking and arguably out of scope for this PR:
* h2 ("Common problems") followed by h3 without any sentence in between.
* not applied h2/h4/h6 pattern (d/k whole page outline of headings to see if can apply)

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/security/?branch=pr-en-us-2939#common-problems